proposals/sar_backscatter.json

```diff
@@ -0,0 +1,127 @@
+{
+    "id": "sar_backscatter",
+    "summary": "Computes backscatter from SAR input",
+    "description": "Computes backscatter from SAR input.\n\nNote that backscatter computation may require instrument specific metadata that is tightly coupled to the original SAR products. As a result, this process may only work in combination with loading data from specific collections, not with general data cubes.",
```

Comment:
This description does not yet mention the mapping of the parameters to sigma0 and gamma0. We can no longer specify these directly, so I believe it would be convenient for the user, and the backend implementor, to at least document what set of parameters corresponds to sigma0 and gamma0.

Reply:
Can someone provide the mapping? I was overwhelmed by the options in the call and can't recall how the mapping is meant to be. @aljacob